### PR TITLE
Get rid of vagrant

### PR DESCRIPTION
**Pull request by Ana06**
We are using Docker now instead.

Closes https://github.com/openSUSE/osem/issues/1213

### CODE REVIEW

**Comment by bear454** [comment]
I haven't been involved in the decision making here, so I'll defer to @hennevogel .

**Comment by hennevogel** [comment]
@bear454 involve yourself in #1213 :wink:

File: config/routes.rb [hunk 1]
```diff
@@ -14,11 +14,6 @@
                path:        'accounts'
   end
 
-  # Use letter_opener_web to open mails in browser (e.g. necessary for Vagrant)
-  if Rails.env.development?
-    mount LetterOpenerWeb::Engine, at: "/letter_opener"
```
Comment (hennevogel) [inline config/routes.rb]: You can also drop the gem from the bundle then.

**Comment by bear454** [comment]
> @bear454 involve yourself in #1213

We're happy with native Heroku at this point, so it's all moot for me. I'd say though, out of courtesy, that someone should check in with the other contributors to the vagrant code and make sure they're not still dependent on it.

**Comment by Ana06** [comment]
@bear454 

> We're happy with native Heroku at this point, so it's all moot for me. I'd say though, out of courtesy, that someone should check in with the other contributors to the vagrant code and make sure they're not still dependent on it.

We can wait a bit before getting this merge, in case someone has a problem with it. But I think changing from vagrant to docker is not a big issue and having to maintain two development environments is extra work.

**Pull request merged**